lib.sh

```diff
@@ -299,6 +300,8 @@ install_popup() {
             exit 1
         fi
     fi
+    # SCRIPT_EXPLAINER is exported in each script, this unsets it.
+    unset SCRIPT_EXPLAINER
```

Comment:
unsetting the variable is not needed when you don't export it which is not needed either to use it in a function